Project4/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <title>Importance of being a generalists</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="first" href="first.js">
```

Comment:
This is not how you reference javascript in your HTML. Instead, you'd want to add a `script` element just before your _closing_ `body` element:
```html
    <script src="first.js"></script>
```